simple-wallet/src/lib.rs

```diff
@@ -249,14 +245,20 @@ where
         let utxos = org_utxos
             .iter()
             .filter(|x| !x.reserved)
-            .map(|x| WeightedUtxo {
-                utxo: BdkUtxo::Local(LocalUtxo {
-                    outpoint: x.outpoint,
-                    txout: x.tx_out.clone(),
-                    keychain: KeychainKind::External,
-                    is_spent: false,
-                }),
-                satisfaction_weight: 107,
+            .map(|x| {
+                WeightedUtxo {
+                    utxo: BdkUtxo::Local(LocalOutput {
+                        outpoint: x.outpoint,
+                        txout: x.tx_out.clone(),
+                        keychain: KeychainKind::External,
+                        is_spent: false,
+                        // @Tibo how do we want to handle this? Store the localoutput in storage?
+                        confirmation_time: ConfirmationTime::unconfirmed(1),
```

Comment:
@Tibo-lg coin selection seems to be the issue for integration tests. Doesn't seem to lead to the unit test though.

Integration tests are very flakey so haven't gotten them to fully pass.

How should we handle the derivation index and conf time? Not sure if it should be included with the `Utxo` type.